android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/authorization/PublicKeysDao.java

```diff
@@ -21,7 +21,7 @@ public class PublicKeysDao extends DbContentProvider<PublicKey> implements Publi
     }
 
     @Override
-    protected PublicKey cursorToEntity(Cursor cursor) {
+    protected PublicKey cursorToEntity(@NonNull Cursor cursor) {
```

Comment:
It looks like the return value from this function is always non-null; can we annotate it as such?